mmstudio/src/main/java/org/micromanager/internal/positionlist/TileCreatorDlg.java

```diff
@@ -364,6 +376,13 @@ public void actionPerformed(ActionEvent arg0) {
       studio.events().registerForEvents(this);
    }
 
+   @Subscribe
+   public void stuttingDown(ShutdownCommencingEvent se) {
```

Comment:
Sorry, it was off by one line (first time I tried GitHub's code commenting feature).
I was referring to the setting of `OVERLAP_PREF` in the profile at shutdown, right below.
(Though I'm also not sure what the `activateAxisTable()` accomplishes here.)